### PR TITLE
Moe/#975 user actiavation link

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: app/test-e2e/verify/walletVerify.spec.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { INV_ETH_EUR_ICBM_HAS_KYC } from "../constants";
+import { tid } from "../utils/selectors";
+import { createAndLoginNewUser } from "../utils/userHelpers";
+import { etoFixtureAddressByName, logoutViaTopRightButton } from "../utils";
+import { goToDashboard } from "../utils/index";
```
Comment (drakkein) [inline app/test-e2e/verify/walletVerify.spec.e2e.ts]: please import from utils without pointing index

Comment (Moejoe90) [reply app/test-e2e/verify/walletVerify.spec.e2e.ts]: hmmm this happened automatically

Comment (drakkein) [reply app/test-e2e/verify/walletVerify.spec.e2e.ts]: Let me guess... VS Code? :D

File: app/test-e2e/verify/walletVerify.spec.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { tid } from "../utils/selectors";
+import { createAndLoginNewUser } from "../utils/userHelpers";
+import { logoutViaTopRightButton } from "../utils";
+import { goToDashboard } from "../utils";
```
Comment (drakkein) [inline app/test-e2e/verify/walletVerify.spec.e2e.ts]: we can group these 2 imports

Comment (Moejoe90) [reply app/test-e2e/verify/walletVerify.spec.e2e.ts]: Good catch

**Pull request merged**